web-app/src/tmp/server.py

```diff
@@ -0,0 +1,46 @@
+from flask import Flask, request, jsonify
+from flask import render_template
+from celery.result import AsyncResult
+from flask_cors import CORS
+
+from .import tasks
+
+import os
+import tempfile
+
+app = Flask(__name__)
+CORS(app)
+
+@app.route("/")
+def index() -> str:
+    return render_template("index.html")
+
+@app.route('/upload', methods=['POST'])
```

Comment:
I guess this file has been superseded by `/views/views.py`?

Reply:
server can be deleted